beacon-chain/chaintest/tests/state-tests/block-processing.yaml

```diff
@@ -17,18 +17,18 @@ test_cases:
       num_slots: 64 # Testing advancing state to exactly slot == SlotsPerEpoch
       deposits:
         - slot: 9223372036854775809
-          amount: 32
-          merkle_index: 0
+          amount: 0
```

Comment:
wait , what is going on here ? why is this 1.
the same below. We always use 32 eth